features/steps/cancle_orderd_steps.py

```diff
@@ -4,72 +4,83 @@
 from selenium.webdriver.support import expected_conditions as EC
 from time import sleep
 
+
 SEARCH_INPUT = (By.XPATH, "//input[@type='search']")
 SEARCH_BTN = (By.CSS_SELECTOR, "div.a-box-inner .help-content")
 BB_SEARCH = (By.ID, 'twotabsearchtextbox')
 SUBMIT = (By.ID, 'nav-search-submit-button')
 F_ITEM = (By.XPATH, "//div[@data-component-type='s-impression-counter']//div[@class='a-price-whole']")
 BST_SELL = (By.CSS_SELECTOR, "#zg_header a")
-COLOR_OPT = (By.CSS_SELECTOR, '#softlinesTwister_feature_div ul')
-COLOR_NAME = (By.CSS_SELECTOR, "#a-autoid-7-announce .imgSwatch")
+
+coffee_block = (By.CSS_SELECTOR, "[data-component-type='s-search-result']")
```

Comment:
Please use the same pattern when creating variables, all the locators should be either upper cased or lower cased, don't mix them. Otherwise your code doesn't look consistent and it's one of the red flags during interviews.

Reply:
h lana thank you for your time.is it aboiut line 15 "coffee_block"? its typo but i will pay attention next time